app/assets/stylesheets/forever_style_guide/modules/_all.scss

```diff
@@ -1,6 +1,7 @@
 @import "nav";
 @import "nav-fixed";
 @import "navbar";
+@import "promo-banner";
```

Comment:
Good point. I think promo_banner is appropriate since conceivably you could implement this anywhere on the page.